deployment/helm/dnsmasqaks/values.yaml

```diff
@@ -0,0 +1,39 @@
+image: ghcr.io/azure-samples/distributed-az-iot-edge/dnsmasqaks:main-41d901d
+
+proxyDnsServer: to_replace
+azureDnsServer: "168.63.129.16" # Azure DNS
```

Comment:
This is documented as 'will not change': https://learn.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16
It is part of the Values.yaml file which does allow to change it if desired through the Helm chart deployment parameters. It's probably OK to set the default value as a fixed value but it can be overwritten if folks want to use their own DNS server.